downstream/modules/navigator/proc-review-ee-tui.adoc

```diff
@@ -0,0 +1,76 @@
+
+[id="proc-review-ee-tui_{context}"]
+
+
+
+= Reviewing {ExecEnvNameStart} from the {Navigator} textual user interface
+
+[role="_abstract"]
+
+You can review your {ExecEnvNameStart} with the {Navigator} textual user interface.
+
+.Prerequisites
+
+* {ExecEnvName}
+
+.Procedure
+
+. Review the {ExecEnvName} included in your {Navigator} configuration.
++
+```
+$ ansible-navigator images
+
+NAME                                                             TAG      EXECUTION ENVIRONMENT       CREATED         SIZE
+0│ansible-automation-platform-20-ee-minimal-rhel8                latest                    True       3 weeks ago     411 MB
+1│ansible-automation-platform-20-ee-supported-rhel8 (primary)    latest                    True       21 hours ago    914 MB
+2│ansible-runner                                                 devel                     True       3 weeks ago     652 MB
+3│network-ee                                                     latest                    True       3 weeks ago     728 MB
+
+
+^f/PgUp page up            ^b/PgDn page down            ↑↓ scroll            esc back            [0-9] goto       :help help
+```
+
+. Type the number of the {ExecEnvName} you want to delve into for more details.
++
+```
+ANSIBLE-AUTOMATION-PLATFORM-20-EE-SUPPORTED-RHEL8:LATEST (PRIMARY)                DESCRIPTION
+0│Image information                                                               Information collected from image inspection
+1│General information                                                             OS and python version information
+2│Ansible version and collections                                                 Information about ansible and ansible collections
+3│Python packages                                                                 Information about python and python packages
+4│Operating system packages                                                       Information about operating system packages
+5│Everything                                                                      All image information
+
+
+^f/PgUp page up              ^b/PgDn page down              ↑↓ scroll            esc back           [0-9] goto        :help help
+```
+
+. Optionally, you can pass in the {ExecEnvName} you want to use. This becomes the primary and is the {ExecEnvName} that {Navigator} uses.
++
+```
+$ ansible-navigator images --eei  quay.io/ansible/network-ee
```

Comment:
avoid network-ee here, only because it's a test-only image.....  maybe we should show these with a mock "internal_enterprise_ee" or something